contracts/ERC721M.sol

```diff
@@ -491,6 +509,15 @@ contract ERC721M is IERC721M, ERC721AQueryable, Ownable, ReentrancyGuard {
         emit Withdraw(value);
     }
 
+    /**
+     * @dev Withdraws ERC-20 funds by owner.
+     */
+    function withdrawERC20() external onlyOwner {
+        uint256 value = IERC20(_mintCurrency).balanceOf(address(this));
```

Comment:
just curious will it revert if _mintCurrency is address(0)? let's unit test this

Reply:
Added an explicit revert and new error - previously, it was returning this error `Error: Transaction reverted: function returned an unexpected amount of data` which is not super informative